fast_api_als/main.py

```diff
@@ -1,9 +1,12 @@
 import time
+import logging
 
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
 from fast_api_als.routers import users, submit_lead, lead_conversion, reinforcement, oem, three_pl, quicksight
 
+logging.basicConfig(format="%(levelname)s: %(asctime)s: %(message)s")
```

Comment:
it is a good practice to maintain a logger.py file in your codebase to set the logger formatting etc . This can be called in different files

Reply:
I have created logger.py in fast_api_als/utils and called it in different files.